engine/apps/mobile_app/tests/test_alert_rendering.py

```diff
@@ -0,0 +1,173 @@
+from unittest.mock import patch
+
+import pytest
+
+from apps.alerts.incident_appearance.templaters.alert_templater import TemplatedAlert
+from apps.mobile_app.alert_rendering import get_push_notification_subtitle, get_push_notification_title
+from apps.mobile_app.backend import MobileAppBackend
+
+MAX_ALERT_TITLE_LENGTH = 200
+
+# this is a dirty hack to get around EXTRA_MESSAGING_BACKENDS being set in settings/ci-test.py
+# we can't simply change the value because 100s of tests fail as they rely on the value being set to a specific value 🫠
+# see where this value is used in the unitest.mock.patch calls down below for more context
```

Comment:
not sure if there's a better approach here?

I wasn't very fond on updating 100+ tests in this PR, that're tightly coupled to the value of `EXTRA_MESSAGING_BACKENDS` in `settings/ci-test.py`